sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1102,46 +1097,85 @@ def get_cmis_dp_init_duration_secs(self, api):
     def get_cmis_dp_deinit_duration_secs(self, api):
         return api.get_datapath_deinit_duration()/1000
 
-    def is_cmis_application_update_required(self, api, channel, speed):
+    def get_cmis_host_lanes_mask(self, api, appl, host_lane_count, subport):
+        """
+        Retrieves mask of active host lanes based on appl, host lane count and subport
+
+        Args:
+            api:
+                XcvrApi object
+            appl:
+                Integer, the transceiver-specific application code
+            host_lane_count:
+                Integer, number of lanes on the host side
+            subport:
+                Integer, subport id of the group which the host lanes belong to (1 based),
```

Comment:
@mihirpat1 subport:
`1-based logical port number of the physical port after breakout`